<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullScreenView</class>
 <widget class="QWidget" name="FullScreenView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muxer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#FullScreenView {
	background-color: rgb(150, 150, 150);

}


/************************************************************
 * Scroll bar 
 * -----------
 *
 * No up,down,left and right arrow buttons.
 * simple dark gray handle with light grey background 
************************************************************/
 QScrollBar:horizontal {
     border: 1px solid grey;
     background: rgb(190, 190, 190);
     height: 15px;
     margin: 0px 0px 0 0px;
 }
 QScrollBar::handle:horizontal {
     background:  rgb(150, 150, 150);
     min-width: 20px;
 }

 QScrollBar::add-line:horizontal {
     width: 0px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:horizontal {
     width: 0px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

 QScrollBar:vertical {
     border: 1px solid grey;
     background:  rgb(190, 190, 190);
     width: 15px;
     margin: 0px 0px 0px 0px;
 }

 QScrollBar::handle:vertical {
     background: rgb(150, 150, 150);
     min-height: 20px;
 }

 QScrollBar::add-line:vertical {
     height: 0px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
     height: 0px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical,
QScrollBar:left-arrow:horizontal, QScrollBar::right-arrow:horizontal  {
      width: 0px;
      height: 0px;
      background: rgb(209, 209, 209);
  }


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal,  
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }</string>
  </property>
  <widget class="QWidget" name="wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#wrapper {
	background-color: rgba(150, 150, 150,220);

}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#toolbar{
	/*background-color:rgba(150, 150, 150, 220);*/
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="searchBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#searchbox{
border: 1px solid grey;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="leSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#leSearch{
	border: 1px solid grey;
	/*border-top-left-radius: 13px;
	border-bottom-left-radius: 13px*/;
	background-color: rgb(217, 217, 217);
	margin: 0px;
	padding-left: 4px;
}</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnSearch{
	border: 1px solid grey;
	border-left: 1px solid grey;
	border-top-right-radius: 13px;
	border-bottom-right-radius: 13px;
	background-color: rgb(191, 191, 191);
	margin:0px;
	padding:0px;
	image: url(:/icons/icons/search_64.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="actions" native="true">
         <property name="styleSheet">
          <string notr="true">#actions{
	border-radius: 10px;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddMusic">
            <property name="toolTip">
             <string>add music to collection</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnAddMusic {
		border: 0px;
	    padding-right: 10px;
	   border-top-left-radius: 10px;
	   border-bottom-left-radius: 10px;
}

#btnAddMusic::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/add_folder.png</normaloff>:/icons/icons/add_folder.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCollection">
            <property name="toolTip">
             <string>show music collection</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnCollection {
	border: 0px;
	margin-left: 0px;
    padding-right: 5px;
    padding-left: 5px;
}

#btnCollection::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/cd_128.png</normaloff>:/icons/icons/cd_128.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSettings">
            <property name="toolTip">
             <string>settings</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnSettings {
   border: 0px;
   padding-left: 10px;
   border-top-right-radius: 10px;
   border-bottom-right-radius: 10px;
}

#btnSettings::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/settings_128.png</normaloff>:/icons/icons/settings_128.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnHalfScreen">
            <property name="toolTip">
             <string>hafl screen</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnHalfScreen {
		border: 0px;
	    padding-right: 2px;
	   border-top-left-radius: 10px;
	   border-bottom-left-radius: 10px;
}

#btnHalfScreen::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/halfscreen.png</normaloff>:/icons/icons/halfscreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFullScreen">
            <property name="toolTip">
             <string>fullscreen</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnFullScreen{

	border: 0px;
    padding-left: 1px;
    padding-right: 1px;
}

#btnFullScreen::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/fullscreen_128.png</normaloff>:/icons/icons/fullscreen_128.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuit">
            <property name="toolTip">
             <string>quit</string>
            </property>
            <property name="styleSheet">
             <string notr="true">#btnQuit{

	border: 0px;
    padding-left: 2px;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

#btnQuit::hover{  background-color: rgb(217, 217, 217);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/icons.qrc">
              <normaloff>:/icons/icons/cancel.png</normaloff>:/icons/icons/cancel.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="body" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#body {
/*padding-top: 50px;*/
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
